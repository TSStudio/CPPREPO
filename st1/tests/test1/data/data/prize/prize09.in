4
liss 78 88 Y
kiss 88 99 N
moon 88 88 N
qwer 99 87 Y