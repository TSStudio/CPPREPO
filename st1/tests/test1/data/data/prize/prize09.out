qwer
14000
32000